### DISCUSSION TITLE
undefined symbol: hb_buffer_set_invisible_glyph

### DISCUSSION THREAD

**Discussion by liuatgit** [ImageMagick/ImageMagick#7114] [General] [answered]
I tried to use the magick AppImage on Centos 7 and got this error message:

./magick: symbol lookup error: /usr/sbin/squashfs-root/usr/bin/../lib/libraqm.so.0: undefined symbol: hb_buffer_set_invisible_glyph

How can I fix this?

**Comment by urban-warrior** [comment, marked as answer]
The AppImage likely is not compatible with Centos 7.  You could also try updating the HarfBuzz library on your system.
